apps/voting/app/src/components/LocalIdentityBadge/LocalIdentityBadge.js

```diff
@@ -2,40 +2,33 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { Badge, IdentityBadge, font } from '@aragon/ui'
-import { LocalIdentityModalContext } from '../LocalIdentityModal/LocalIdentityModalManager'
 import { IdentityContext } from '../IdentityManager/IdentityManager'
 
-const LocalIdentityBadge = ({ address, ...props }) => {
-  const { resolve } = React.useContext(IdentityContext)
-  const { showLocalIdentityModal, updates$ } = React.useContext(
-    LocalIdentityModalContext
-  )
-  const [label, setLabel] = React.useState()
-  const handleResolve = async () => {
-    try {
-      const { name = null } = await resolve(address)
-      setLabel(name)
-    } catch (e) {
-      // address does not ressolve to identity
-    }
-  }
-  const handleClick = () => {
-    showLocalIdentityModal(address)
-      .then(handleResolve)
-      .catch(e => {
-        /* user cancelled modify intent */
-      })
+function useIdentity(address) {
```

Comment:
Since this is the only place it is being used it makes sense to define it here, eventually we could add this to the new [`aragonApi`](https://github.com/aragon/aragon.js/pull/259).